__tests__/app.test.js

```diff
@@ -183,6 +183,42 @@ describe("GET Tests", () => {
         });
     });
   });
+  describe("GET Users", () => {
+    test("should return a Status Code : 200", () => {
+      return supertest(app).get("/api/users").expect(200);
+    });
+    test("should return an array of users", () => {
+      return supertest(app)
+        .get("/api/users")
+        .expect(200)
+        .then((response) => {
+          expect(response.body.users).toBeInstanceOf(Array);
```

Comment:
Generally there's no need to assert that the response is an array as later we are using a forEach on the response therefore proving that it is an array